### PR TITLE
Use pf3 ExpandCollapse react component in the VM Details card

### PR DESCRIPTION
**Pull request by hstastna**
Fixes: https://github.com/oVirt/ovirt-web-ui/issues/1296
**Depends on:** https://github.com/oVirt/ovirt-web-ui/pull/1279

In this PR, I used the pf3 react `ExpandCollapse` component instead of our implementation of the expand/collapse `ExpandCollapseSection` component in the VM Details card. Also the css style of the pf `ExpandCollapse` was slightly edited to achieve the expected look (same as before, extra space above the _Advanced Options_ and above their content, proper align of the expand/collapse icon (^ V) with the rest of the component).
Also the `ExpandCollapseSection` in the _Edit NIC_ dialog (displayed while creating/editing a NIC of an existing VM) was replaced, too.

**Before:**
VM Details card:
before
Edit NIC dialog:
nic_before

**After:** 
after
nic_after

Note that these screenshots here demonstrate that no any visual change should be/was made with changes in this PR.

### CODE REVIEW

File: src/components/VmDetails/cards/DetailsCard/index.js [hunk 1]
```diff
@@ -839,9 +839,9 @@ class DetailsCard extends React.Component {
               </Row>
             </Grid>
             {/* Advanced options */}
-            { isFullEdit && <ExpandCollapseSection id={`${idPrefix}-advanced-options`} sectionHeader={msg.advancedOptions()}>
+            { isFullEdit && <ExpandCollapse id={`${idPrefix}-advanced-options`} className={style['adv-options-divider']} textCollapsed={msg.advancedOptions()} textExpanded={msg.advancedOptions()}>
```
Comment (sjd78) [inline src/components/VmDetails/cards/DetailsCard/index.js:842]: Can the CSS manipulation to make the PF3 `ExpandCollapse` component look more like our `ExpandCollapseSection` be done by modifying the PF3 css?

Those mods can go in `index-nomodules.css`.  That would be preferred so the `adv-options-divider` doesn't need to be specified on a component each time the `ExpandCollapse` is used.

Relevant classes:
  - `expand-collapse-pf`
  - `expand-collapse-pf-link-container`
  - `expand-collapse-pf-separator`

**Review by sjd78 — DISMISSED** [review]

File: src/components/VmDetails/cards/DetailsCard/index.js (hunk 1 above)
Comment (sjd78) [reply src/components/VmDetails/cards/DetailsCard/index.js:842]: You can see them in use in the component over at: https://github.com/patternfly/patternfly-react/blob/patternfly-react%402.39.16/packages/patternfly-react/src/components/ExpandCollapse/ExpandCollapse.js

Comment (hstastna) [reply src/components/VmDetails/cards/DetailsCard/index.js:842]: https://github.com/oVirt/ovirt-web-ui/issues/1264#issuecomment-696258357

**Review by bamsalem — APPROVED** [review]
except @sjd78 inputs, LGTM

**Review by sjd78 — APPROVED** [review]
LGTM - Much cleaner with the CSS class tweaks

**Pull request merged**